### ISSUE TITLE
Allow displaying artists also in a grid, not only a list, using pics from last.fm database

### ISSUE THREAD

**Issue by marek-lach** [navidrome/navidrome#1535]
Currently for sorting by artists, only a list view is possible.
![navidome_artists_can_only_display_as_list](https://user-images.githubusercontent.com/45913977/147261163-7ec7ee8d-b4ad-4290-ba2b-3e136f849727.png)

That is perfectly fine, but an option to also (if preferred by a user) show sort by artist as a grid would be cool:

![show_artist_pictures_according_to_lastfm](https://user-images.githubusercontent.com/45913977/147261335-7c9065fa-8a6c-46e0-9aa7-77a1915d5fca.png)

Last.fm can be used to scrub the artist pictures for such a grid view.

**Comment by Dnouv** [comment]
Thank you so much for the suggestion, @marek-lach; the idea sounds good; maybe this could be incorporated in future releases; however, please don't hold your breath. Let's hear thoughts of @certuna or @metalheim on the enhancement.

If anyone would like to work on these contributions are welcome.
Please feel free to comment. Thank you!

**Comment by metalheim** [comment]
I'm pretty agnostic to this addition. Personally, i probably won't use it but it also doesn't harm to have it.

Some more thoughts:
- This probably only makes sense if external integrations are activated - maybe hide it if theyre not active? Otherwise you wouldn't see any images anyways. (Unless, in future there is a "local lookup" agent for artist image #394 )

- Afaik, the artist images get cached, so it shouldn't put too much load on the network and external APIs. Even less so with Lazy Loading and infinite scrolling

**Comment by tcurdt** [comment]
With the improved artist image support this would be really nice to have.
But it seems like there is not enough interest to make this happen?

**Comment by deluan** [comment]
Yes, there is interest, but I'm not planning to spend too much effort with the current UI, as I'm having to rewrite it to use the new RA 4.

**Comment by tetebueno** [comment]
I noted that AudioDB (https://www.theaudiodb.com/api_guide.php) is not being considered as a source for images. This API is used in Jellyfin and I've found it to provide better images (compared to Spotify) in terms of resolution, but also in the fact that it provides logos and banners for artists, this allows for a more rich UI. For example:

![image](https://user-images.githubusercontent.com/9064236/223909103-361082d3-0620-424f-97ab-3fd04fb32ac7.png)

Hope this brings more help than noise to the matter.

**Comment by deluan** [comment]
> Maximum 1 call per 2 seconds on all API methods. 

$3 for a month for a full private API key and higher limits.

Even though I really think they deserve being paid for their work, I prefer to implement image sources that are free for the end user. 

Is it the only source for images in Jellyfin? Not sure how they were able to use the service with such low limit.

**Comment by tetebueno** [comment]
Oh, I didn't see that (my color blindness gave me away). Sorry.

I know it fetches stuff from different sources, including [AudioDB](https://github.com/jellyfin/jellyfin/tree/master/MediaBrowser.Providers/Plugins/AudioDb), [Cover Art Archive](https://github.com/jellyfin/jellyfin-plugin-coverartarchive) or [MusicBrainz](https://github.com/jellyfin/jellyfin-plugin-musicbrainz). But, as far as I can tell -after examining the code, images are being taken from AudioDB. It doesn't make much sense considering those limits.

I'll chime in if I find out more about it.

**Comment by tcurdt** [comment]
Un-stale. I hope this will be part of of the UI rewrite.

**Comment by tcurdt** [comment]
Please un-stale.

**Comment by Andrew-H2O** [comment]
I would appreciate this enhancement too.

**Comment by tcurdt** [comment]
Maybe worth having a look at https://github.com/jeffvli/feishin/ it's an incredible client that almost ticks all boxes for me.
Also does the artist grid.

(no affiliation - just a fan)

**Comment by stapelberg** [comment]
please un-stale

**Comment by Metal-Snake** [comment]
I would love this too.

**Comment by Metal-Snake** [comment]
Can this be reopened?
